addons/base_setup/res_config_view.xml

```diff
@@ -169,18 +169,8 @@
                             <div name="Sale Features"/>
                         </group>
                     </div>
-                    <div name="config_sale">
-                        <field name="module_crm" invisible="1"/>
-                        <separator string="Emails Integration" attrs="{'invisible': [('module_crm','=',False)]}"/>
-                        <p name="config_email_integration" attrs="{'invisible': [('module_crm','=',False)]}">
-                            Odoo allows to automatically create leads (or others documents)
-                            from incoming emails. You can automatically synchronize emails with Odoo
-                            using regular POP/IMAP accounts, using a direct email integration script for your
-                            email server, or by manually pushing emails to Odoo using specific
-                            plugins for your preferred email application.
-                        </p>
-                    </div>
-                    <div name="config_other"/>
+                    <div name="config_sale"/>
```

Comment:
I tried without the void divs, and if I install point_of_sale, and then crm, all the divs are messed up. 
See : 
https://docs.google.com/file/d/0Bz6OpV442af-UWlFQTJLMzhaVGM/edit?usp=drivesdk
https://docs.google.com/file/d/0Bz6OpV442af-TkswMnUwRGM3MGs/edit?usp=drivesdk

So I think we should not do it.